interface/memory.h

```diff
@@ -32,6 +32,8 @@ void memoryInit();
 
 bool memorySyslink(struct syslinkPacket *pk);
 
+bool memoryHasBcAiDeck(void);
```

Comment:
It might be a bit more clean to have a generic function there as in:

``` c
#define MEMORY_BITCRAZE_VID 0xBC
#define MEMORY_AIDECK_PID 0x12

bool memoryHasDeck(uint8_t vid, uint8_t pid);
```

And call is with with:
``` c
memoryHasDeck(MEMORY_BITCRAZE_VID, MEMORY_AIDECK_PID);
```

Reply:
Good idea, fixed in latest commit